### PR TITLE
Add CVE-2020-10738.

### PR DESCRIPTION
**Pull request by guitos**

### CODE REVIEW

File: 2020/10xxx/CVE-2020-10738.json [hunk 1]
```diff
@@ -4,15 +4,87 @@
...
+                                        },
+                                        {
+                                            "version_value": "3.7 to 3.7.5"
+                                        },
+                                        {
+                                            "version_value": "3.6 to 3.6.9"
+                                        },
+                                        {
+                                            "version_value": "3.5 to 3.5.11"
+                                        }
```
Comment (iamleot) [inline 2020/10xxx/CVE-2020-10738.json:31]: Please mention fixed version, and adjust the 3.5.x pattern to match also unsupported versions:

 - `>=3.8,<3.8.3`
 - `>=3.7,<3.7.6`
 - `>=3.6,<3.6.10`
 - `<3.5.12`

File: 2020/10xxx/CVE-2020-10738.json [hunk 2]
```diff
@@ -4,15 +4,87 @@
...
+                "refsource": "CONFIRM"
+            }
+        ]
     },
     "description": {
         "description_data": [
             {
                 "lang": "eng",
-                "value": "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem. When the candidate has been publicized, the details for this candidate will be provided."
+                "value": "A flaw was found in Moodle versions 3.8 to 3.8.2, 3.7 to 3.7.5, 3.6 to 3.6.9, 3.5 to 3.5.11 and earlier unsupported versions. It was possible to create a SCORM package in such a way that when added to a course, it could be interacted with via web services in order to achieve remote code execution."
```
Comment (iamleot) [inline 2020/10xxx/CVE-2020-10738.json:76]: Same as above, please mention fixed version so readers known that there are versions released with the fix to this CVE.

**Comment by guitos** [comment]
Updated affected entries with earlier unsupported versions and add fixed versions in the description.

**Comment by iamleot** [comment]
Thanks!

**Pull request merged**